applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -120,6 +120,23 @@ class TrilinosBlockBuilderAndSolver
     ///@name Life Cycle
     ///@{
 
+    /**
+     * @brief Default constructor with Parameters
+     * @param pNewLinearSystemSolver The linear solver for the system of equations
+     * @param ThisParameters The configuration parameters
+     */
+    explicit TrilinosBlockBuilderAndSolver(
```

Comment:
Nope, I took it from the base class. I didn't notice a params constructor on your branch, otherwise I would have used that one...

Will try to merge, no sense in duplicating efforts.

Reply:
this needs to be exported to python by the way

Reply:
I think is exported on my PR, if not should be

Reply:
Yes, the commit: https://github.com/KratosMultiphysics/Kratos/commit/ba77c3ad5d20b81fdf7286567bda51f40b85f15d, you can cherry-pick